### PR TITLE
Setting up server

### PR DESCRIPTION
**Pull request by 95Rawan**

### CODE REVIEW

**Review by SajaLahaleeh — APPROVED** [review]

**Review by someyaaltous — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,71 @@
+# Hebron Rest:
 
+## Description:
+
+A website that shows the user the four most expensive paintings in the world.
+When the user clicks on one of them, it shows some details about them. It also
+allows you to add a painting if you want, with a list of details about it using
+a form.
```
Comment (jema28) [inline README.md]: 'It also allows you to add a painting.'

File: README.md [hunk 2]
```diff
@@ -1 +1,71 @@
...
+    - 2.png
+    - 3.png
+    - AbuMazin.png
+    - TabonAlreef.png
+    - KFC.png
+    - RozaChrunchy.png
+    - Q-Cany.png
+    - Segafredo.png
+    - logo.png
+  - index.html
```
Comment (jema28) [inline README.md]: There shouldn't be any index.html in public, we're using handlebars to replace this.

File: README.md [hunk 3]
```diff
@@ -1 +1,71 @@
...
+  - style.css
+  - index.js
```
Comment (jema28) [inline README.md]: No js files here either.

File: README.md [hunk 4]
```diff
@@ -1 +1,71 @@
...
+
+- src:
+- controllers
+- views:
+  - partials: headers.hbs
+  - layouts:
+    - main.hbs
+  - database:
```
Comment (jema28) [inline README.md]: The database shouldn't be in the views. Move this to root of src.

File: README.md [hunk 5]
```diff
@@ -1 +1,71 @@
...
+    - db_build.js
+    - db_build.sql
+    - db_connection.js
+  - queries:
```
Comment (jema28) [inline README.md]: Put queries inside database folder.

File: src/app.js [hunk 6]
```diff
@@ -0,0 +1,26 @@
+const express = require("express");
+const path = require('path');
+const exphbas = require("express-handlebars");
+const app = express();
+app.use(router)
+
+
+app.use(express.static(path.join(__dirname, '..', 'public')));
+
+app.set("views", path.join(__dirname, "views"));
```
Comment (jema28) [inline src/app.js:10]: Please add disable x powered by. You're also missing (`app.use(express.json())`) and urlencoded.

Comment (jema28) [reply src/app.js:10]: Double check the first code along.

**Review by jema28 — CHANGES_REQUESTED** [review]

File: src/app.js [hunk 7]
```diff
@@ -0,0 +1,26 @@
+const express = require("express");
+const path = require('path');
+const exphbas = require("express-handlebars");
+const app = express();
+app.use(router)
```
Comment (jema28) [inline src/app.js]: This should happen after everything else in this file (before exports).

**Review by jema28 — CHANGES_REQUESTED** [review]

**Comment by jema28** [comment]
Please use the normal eslint setup for this file (not wesbos as it's for react). Please set this app as soon as possible.

**Review by jema28 — APPROVED** [review]

**Pull request merged**